src/frontends/pytorch/src/op/pythonop.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+#include "translate_session.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_pythonop(NodeContext& context) {
+    auto decoder = context.get_decoder();
+    FRONT_END_OP_CONVERSION_CHECK(decoder->get_subgraph_size() == 1,
+                                  "PythonOp must have 1 subgraph to be able to translate it to OV.");
+    auto body = context.convert_subgraph(0);
+    auto session = context.get_session();
+
+    std::map<size_t, ParameterVector> inputs_map;
+    for (const auto& param : body->get_parameters()) {
+        auto tensor_idx = session->decode_tensor_name(param->output(0));
+        if (!inputs_map.count(tensor_idx)) {
+            inputs_map[tensor_idx] = {param};
+        } else {
+            inputs_map[tensor_idx].push_back(param);
```

Comment:
Is it not expected and should throw exception? If more then 1 parameter has the same input?

Reply:
Added `FRONT_END_OP_CONVERSION_CHECK` to not allow more than 1 parameter with the same input.